app/src/test/java/ru/vssemikoz/newsfeed/newsfeed/NewsFeedPresenterTest.java

```diff
@@ -84,53 +90,62 @@ private String generateRandomString(int stringSize) {
     }
 
     @Test
-    public void verifyUpdateActualNews_ShowProgressBarIsCalled() {
+    public void verifyUpdateActualNews_UpdateStorageUseCaseIsCalled() {
+        when(updateStorageUseCase.run(any())).thenReturn(Single.just(exampleNewsList));
+        when(getFilteredNewsUseCase.run(any())).thenReturn(Single.just(exampleNewsList));
         presenter.updateActualNews();
-        verify(view).showProgressBar();
+        verify(updateStorageUseCase).run(any());
     }
 
     @Test
-    public void verifyUpdateActualNews_UpdateStorageUseCaseIsCalled() {
-        presenter.updateActualNews();
-        verify(updateStorageUseCase).run(any());
+    public void verifyUpdateActualNews_UpdateStorageUseCaseThrowError() {
```

Comment:
This test is not needed since you're doing nothing here. You mocked a usecase call to call it and checked that it did exactly what you've mocked. It's a test to test a Mockito. You should remove it.